PROVISIONING-METAL.md

```diff
@@ -106,10 +107,46 @@ Please keep in mind that when using static addresses, DNS information must be su
   * `via` (IP address): Gateway IP address.  If no gateway is provided, a scope of `link` is assumed.
   * `route-metric` (integer): Relative route priority.
 
-Example `net.toml` with comments:
+Version `3` adds in support for bonding and vlan tagging. 
+The support is limited to mode `1` (`active-backup`) for [bonding](https://www.kernel.org/doc/Documentation/networking/bonding.txt).
+Future support may include other bonding options - pull requests are welcome! 
+Version `3` also adds the concept of network devices vs network interfaces.
```

Comment:
Here we say `network devices vs. network interfaces`, but the following sentence refers to an interface as a type of device.

Perhaps we should say something like `Version 3 adds the concept of virtual network devices in addition to interfaces`